server/tools/hatohol-db-initiator.in

```diff
@@ -180,7 +180,9 @@ def start(args):
     create_hatohol_tables(args)
 
     try:
-        for sql_file in os.listdir(args.sql_dir):
+        sql_files = os.listdir(args.sql_dir)
+        sql_files.sort()
```

Comment:
Oh, Python's list.sort() method replaces elements in-place!
...No kidding? :cold_sweat: